infrastructure/docker-images/build-browser-test.sh

```diff
@@ -11,7 +11,7 @@ fi
 tmpdir=$(mktemp -d)
 trap "cd /; rm -rf ${tmpdir}" EXIT
 cat > ${tmpdir}/Dockerfile <<EOF
-FROM node:${node_version}-buster
+FROM node:${node_version}-bookworm
```

Comment:
I wonder if we could do something similar here to the `{node_version}` to avoid doing `s/old/new/` in so many places? 
just a thought :)